engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -717,9 +718,8 @@ public void postSave() { }
     public void shutdown() { }
 
     @Command(shortDescription = "Debugging command for DAG.", requiredPermission = PermissionManager.NO_PERMISSION)
-    public void dagNodeCommand(@CommandParam("nodeUri") final String nodeUri, @CommandParam("command") final String command, @CommandParam("arg1") final String arg1) {
-        // TODO: Convert arg1 to args[]
-        Node node = renderGraph.findNode(nodeUri);
-        node.handleCommand(command, arg1);
+    public void dagNodeCommand(@CommandParam("nodeUri") final String nodeUri, @CommandParam("command") final String command, @CommandParam(value= "args") final String args[]) {
+        Node node = renderGraph.findNode(new SimpleUri(nodeUri));
+        node.handleCommand(command, args);
```

Comment:
A couple of things:
1. `args` -> `arguments`
2. Would it be possible to use varargs instead of an array? I.e. `final String... arguments`?